docs/resources/server/datamaps/builtin.md

```diff
@@ -0,0 +1,169 @@
+# Built-In Data Maps
+
+NeoForge provides various built-in [data maps][datamap] for common use cases, replacing hardcoded vanilla fields. Vanilla values are shipped by data map files in NeoForge, so there is no functional difference to the player.
+
+## `neoforge:compostables`
+
+Allows configuring composter values, as a replacement for `ComposterBlock.COMPOSTABLES` (which is now ignored). This data map is located at `neoforge/data_maps/item/compostables.json` and its objects have the following structure:
+
+```json5
+{
+  // A 0 to 1 (inclusive) float representing the chance that the item will update the level of the composter
+  "chance": 1
+}
+```
+
+Example:
+
+```json5
+{
+  "values": {
+    // Give acacia logs a 50% chance that they will fill a composter
+    "minecraft:acacia_log": {
+      "chance": 0.5
+    }
+  }
+}
+```
+
+## `neoforge:furnace_fuels`
+
+Allows configuring item burn times. Note that other in-code methods like `IItemExtension#getBurnTime` will take priority over the data map, so it is recommended that you use the data map for simple, static burn times so that users can configure them. This data map is located at `neoforge/data_maps/item/furnace_fuels.json` and its objects have the following structure:
+
+```json5
+{
+  // A positive integer representing the item's burn time in ticks
+  "burn_time": 1000
+}
+```
+
+Example:
+
+```json5
+{
+  "values": {
+    // Give anvils a 2 seconds burn time
+    "minecraft:anvil": {
+      "burn_time": 40
+    }
+  }
+}
+```
+
```

Comment:
Please add back the note about `IItemExtension#getBurnTime()` overruling the datamap